### PR TITLE
User guide: mention Windows 11

### LINKED ISSUE #12675: User guide: begin mentioning Windows 11
Hi,

This is mostly for Quentin:

### Is your feature request related to a problem? Please describe.
At the moment user guide and other documentation notes Windows 10 as the latest supported Windows release. With Windows 11 being finalized, along iwth NVDA 2021.2 code moving to beta soon (with foundations to support Windows 11 included), it makes sense to start mentioning Windows 11 in user-facing documentation such as the user guide.

### Describe the solution you'd like
At least for the user guide, edit the following sections:

* Supported Windows releases
* Windows 10 OCR
* Windows OneCore voices
* Other parts where Windows 10 mentioned or implied

### Describe alternatives you've considered
None

### Additional context
This assumes NVDA 2021.2 stable version will be released around the time Windows 11 will be released to the general public.

Thanks.

### PR DESCRIPTION
**Pull request by josephsl**
### Link to issue number:
Closes #12675 

### Summary of the issue:
According to the user guide, latest supported Windows release is Windows 10. As Windows 11 is on the horizon, it makes sense to mention the new operating system in approrpate places throughout the user guide.

### Description of how this pull request fixes the issue:
Edited the following contents:
* Supported operating systems (add a note that Windows 11 does not support 32-bit systems but runs 32-bit apps such as NVDA)
* Windows 10 OCR (mention Windows 11 indirectly)
* Windows OneCore voices (indirectly mention Windows 11)
* Other places where Windows 10 is mentioned

### Testing strategy:
User guide generation, verifying information on Windows 11 (build 22000).

### Known issues with pull request:
Few thoughts:
* Should limitations with screen curtain be mentioned?
* Should Windows 10 OCR be renamed? If so, a new issue/PR pair should be used.

### Change log entries:
None (this is user guide)

### Code Review Checklist:

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual testing.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers

### CODE REVIEW

**Comment by josephsl** [comment]
Hi,

Note: PR based on beta branch as master branch was merged into it last week.

Thanks.

File: user_docs/en/userGuide.t2t
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:1839]: Might as well make the capitalisation consistent here
```suggestion
 For the most recent versions of Microsoft Office 2016/365 running on Windows 10 and later, UI Automation support is complete enough to provide access to Microsoft Word documents almost equal to NVDA's existing Microsoft Word support, with the added advantage that responsiveness is majorly increased.
```

File: user_docs/en/userGuide.t2t
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:69]: I have a concern that specifying this might confuse some readers, as the following is still a true statement:

```all 32-bit and 64-bit editions of Windows 7, ..., Windows 11, and all .. ```

File: user_docs/en/userGuide.t2t
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:72]: Do NVDA users need to know this, considering we have said that Windows 11 is supported? I don't think the NVDA user guide is a good source of truth for Windows 11 functionality.

**Review by seanbudd — COMMENTED** [review]
Thanks @josephsl, I left some notes on this PR but I would wait to hear what @Qchristensen and others think before changing anything based on my suggestions. I agree that this should be included in beta.

**Comment by josephsl** [comment]
Hi, for the first point, capitalization issue was present before this PR, but it might be best to address it now. As for the second and third point, I think it is better to at least provide truths about Windows 11 requiring 64-bit systems because that’s what Microsoft notes in its specs (one of the first documentation users would read about screen readers is user guide, and if we say NVDA supports 32-bit and 64-bit Windows releases, we might as well provide more accurate info by including a note about Windows 11 here; @BriTechGuy, any thoughts?). Thanks.

**Comment by britechguy** [comment]
My general thoughts are that:

1.  As much as possible, drop references to Windows versions, directly, at all.  There needs to be an understanding, in the age of Windows as a Service, and with Windows 7 now being dead, that "currently supported versions of Windows" is what Windows, not otherwise categorized, means.  With Microsoft ditching it's, "Windows 10 is the final version of Windows," stance, we have no idea how long it will be until Windows 12 (or something else, will follow).  It makes sense to me to avoid version specification to the greatest extent possible.
2. Bit depth of Windows is very rapidly becoming irrelevant.  It has been ages since 32-bit hardware was produced.  32-bit software routinely runs under 64-bit hardware and, as you note, very soon even 32-bit hardware will not be supported by the very latest version of Windows.

There needs to be an acknowledgement that being too specific with regard to Windows versions makes documentation a nightmare.  One could put something either at the beginning, or end, of the documentation that says something like, "Windows is construed to mean Windows 10 (32 or 64-bit), Windows 11 (64-bit only), and any successors."

The main body of the documentation should strive to be as broadly applicable as possible, with specifics defined once, wherever is felt appropriate.

**Comment by seanbudd** [comment]
I would generally support @britechguy's changes here, but perhaps they should be in a follow up PR as this might be contentious. 

We do need to be specific with versions we support somewhere, as it can be why someone may or may not upgrade or buy a device. 

I would support changing this documentation to be more general, but maybe by specifically defining "Legacy Windows (<10)" and "Modern Windows (10+)". That way we can minimise changes for support being dropped for Legacy Windows or new Windows being supported. Most of the changes here used groups of "Modern Windows" or "Legacy Windows". Windows 7 support will be dropped with the next Python upgrade. While these changes occur fairly slowly, it might be helpful for readers to be able to track the changes consistently all in one place. 

I somewhat agree with @britechguy, I'm not sure the value is in specifying the Bit depth of in Windows, other than confirming that all available versions are supported. However, I'm also hesitant to change this, as 32bit support is getting worse as we progress into the future. For example, the screen curtain does not currently work on Windows 7 due to the Windows API not fully supporting WOW64 (it's possible this is the cause of the Windows 11 problem too, I'm currently investigating this). In fact, in general the API we use for screen curtain is (now) no longer support for WOW64: ["The Magnification API is not supported under WOW64; that is, a 32-bit magnifier application will not run correctly on 64-bit Windows."](https://docs.microsoft.com/en-us/windows/win32/winauto/magapi/entry-magapi-sdk). We currently cannot update to python3.8+ due to a bug with 32bit support.

**Comment by britechguy** [comment]
@seanbudd wrote, in part, " Windows 7 support will be dropped with the next Python upgrade."

If I am not mistaken, this will be a part of 2022.1, so in terms of development pipeline that's the very near future.  I know that Freedom Scientific's changes in the same time frame will even drop support for Windows 8.x.  Essentially, everything we consider to be Windows Legacy is going to be completely out of support in the clearly foreseeable and not-too-distant future.  I don't know when NVDA might have Windows 8.x fall out of support.  Might it be even before January 2023?

Perhaps another PR is needed, I'm not arguing about that, or anything, really.  But the days of Windows Legacy are numbered, and very, very close to their end.  Focus should be more on how these references will be purged going forward rather than maintained.

We are so far in to the Windows 10 era (and I include the forthcoming Windows 11 as part of that) and so near the end of all that came before that planning for the demise of all that came before really does need "front and center" consideration.

**Comment by josephsl** [comment]
Hi, regarding Python upgrades, we also need to balance that with implications for things such as SAPI4 support; earlier this year we did experiment with Python 3.8 but ran into problems. If I’m to advocate Python upgrade, I advise 3.0 at the least, or better yet, 3.10, but then lack of wxPython development is going to cause headaches for a while. Thanks.

**Comment by seanbudd** [comment]
@britechguy - the upgrade to Python3.8+ is currently blocked by the issue Joseph described. Python 3.8, which is the last version that supports Windows 7, has an EOL of 14 Oct 2024. So we will drop Windows 7 support sometime before then, but it may not be in 2022.1

**Comment by britechguy** [comment]
@seanbudd,

Thanks for the clarification.

If there's a choice between positioning NVDA to move "boldly into the future" that breaks backward compatibility with Windows 7, and maintaining that compatibility, I fall firmly in support of the former.

Any compatibility, of anything, with Windows 7 at this point in time should not be a priority and should only occur if it is a an accidental compatibility maintenance.  There are lots of things that remain compatible because the next major update that would break it has not occurred yet.  But Windows 7 is dead, and no hoops should be jumped through to maintain backward compatibility.

**Comment by josephsl** [comment]
Hi, it isn’t just Windows 7 – we might as well take this time to end support for SAPI4, knowing that people will complain about breaking speech synthesizers. A similar concern was raised when NV Access dropped support for Adobe Flash. Given that Windows 8.x and Server 2012/2012 R2 will exit support in 2023, I think the best target would be 2023.1, at which point folks would have moved onto Windows 10 and Windows 11 would have been tested in the wild for at least a year. Thanks.

**Review by seanbudd — DISMISSED** [review]

File: user_docs/en/userGuide.t2t
Comment (leonardder) [inline user_docs/en/userGuide.t2t:812]: Should we rename this to be more generic?

```suggestion
++ Windows OCR ++[Win10Ocr]
```

File: user_docs/en/userGuide.t2t
Comment (leonardder) [inline user_docs/en/userGuide.t2t:2300]:
```suggestion
Windows 10 and later includes voices known as "OneCore" or "mobile" voices.
```

**Comment by josephsl** [comment]
Hi, I think it would be best to rename Windows 10 OCR to Windows OCR or something sooner than later (this must be done for both docs and UI). Thanks.

**Review by seanbudd — APPROVED** [review]

**Comment by valiant8086** [comment]
Hi.


I'm tempted to complain about breaking speech synthesizers. Is there a 
distinct advantage to removing SAPI4 support from NVDA, and are you 
aware that SAPI4 tts engines like True Voice install and work well in 
Windows 10? I have not tried in Windows 11. True Voice is the best 
example I can come up with. It's actually a good sounding tts and is 
well liked at least by people I've ever asked. It reminds me distantly 
of the BDL voice from RH Voice. Only other one we'd care about in 
English anyway to my knowledge would be Mary Mike and Sam which are 
better quality voices than their SAPI5 counterparts at least Sam is. 
There's also Flextalk but I'm not even sure that's supposed to be free 
and it's not very good so I'm ok with not being able to use it.

**Comment by josephsl** [comment]
Hi, SAPI4 issue is a separate matter from this one – we (NVDA contributors and NV Access) learned that SAPI4 support is affected when we use Python 3.8 or later. Besides, the fact that certain Windows API’s are deemed “unstable” for WoW64 apps such as NVDA, coupled with attempts by Microsoft to get folks to use modern technologies are indication that the days of SAPI4 support is slowly coming to a close (this is my comment on it at this point, as the final decision rests with NV Access and market conditions). Thanks.

**Pull request merged**
